app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,19 @@
+<footer class="footer">
+  <div>
+    <div class="footer__heading">Question or feedback</div>
+    <p>Spotted a typo? Something missing?</p>
+    <p>Please <a href="https://github.com/buildkite/docs/issues">open an issue</a> or <a href="<%= open_source_url %>">contribute an update</a>.</p>
+  </div>
+  <div>
+    <div class="footer__heading">Get help</div>
```

Comment:
These headings didn't read clearly to me, now that there are two.

I saw: "go to docs repo issues for feedback [on the product]", to get help go to the community Slack.

Could these be re-written so the stuff on the left is for docs-related feedback, and the path on the right for product-related? ie -- I want to encourage feedback in the Slack channel, too!

Reply:
Hi @meiqimichelle, for context and where my brain's at 🧠 🌈… on top of feedback, another use for the right side product-related area is support. So when customers need to get more help but Docs didn't cover enough content for it.

I like @karensawrey's new heading "Reach out", as it's inclusive of product-related feedback and getting support via email.

Lastly (out of scope for this PR), when we have the "rate how helpful this article is" feature, I'm thinking of including it in this area too.

Seems like we're happy with @karensawrey 's suggestion further down the thread, I'll mark this conversation as resolved. Feel free to raise other ideas as needed.